pkg/apis/monitoring/v1alpha1/prometheus_operator_types.go

```diff
@@ -26,3 +27,37 @@ type PrometheusOperatorSpec struct {
 	// Image of config-reloader sidecar container
 	ConfigmapReloaderImage string `json:"configmapReloaderImage,omitempty"`
 }
+
+type PrometheusProvisioning struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec PrometheusProvisioningSpec `json:"spec"`
+}
+
+// PrometheusProvisioningSpec extends PrometheusSpec from prometheus-operator by adding fields necessary for deployment
+type PrometheusProvisioningSpec struct {
+	prom.PrometheusSpec `json:",inline"`
+	// Namespace to deploy prometheus statefulset in.
+	Namespace string `json:"namespace,omitempty"`
+	// Namespaces to allowed to be watched by prometheus
+	// +optional
+	// kubebuilder:default=["default","kube-system"] // TODO: how can we include a field value that needs to be defined by user? // FIXME(paulfantom): controller-gen doesn't like defaulting arrays
+	Namespaces []string `json:"namespaces,omitempty"`
+}
+
+type AlertmanagerProvisioning struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec AlertmanagerProvisioningSpec `json:"spec"`
+}
+
+type AlertmanagerProvisioningSpec struct {
+	prom.AlertmanagerSpec `json:",inline"`
+	// Namespace to deploy alertmanager statefulset in.
+	Namespace string `json:"namespace,omitempty"`
+	// Namespaces to allowed to be watched by alertmanager
+	// +optional
+	Config map[string]string `json:"config,omitempty"`
```

Comment:
The comment and field don't seem to match

Reply:
Copy-paste issue. Fixed now.